bibtexparser/middlewares/names.py

```diff
@@ -7,6 +7,14 @@
 import dataclasses
 from typing import List, Tuple
 
+# Literal is only available in Python 3.8+
+try:
```

Comment:
In the meantime, python 3.7 has reached EOL. 

I have just updated the CI on master to not use 3.7 anymore, thus feel free to remove your fallback.